BrainStation/BrainStation.h

```diff
@@ -20,8 +20,9 @@
 #include <BrainDelegate.h>
 
 #define GAME_OVER 1
-#define CTRL_CELLS 5
+#define CTRL_CELLS 8
```

Comment:
The explanation of that is here: [README.md - Cells Obs](https://github.com/brain-labs/brainstation/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R48)

Before, we were using only 2 cells for controlling an object plus 3 for keeping data, but I decided to increase the controlling cells to 5 and keep the 3 ones for data (therefore 8 cells in total).. I think that with 5 cells we can keep more information about an object.. There was also better because of the new operator `^`, which requires at least one cell for jumping around in the turing tape.